src/System.Runtime.CompilerServices.VisualC/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
I don't think I can do that, at least for netfx:

`msbuild /T:BuildAndTest /P:TargetGroup=netfx`

```
RunTestsForProject:
  C:\Users\hugh\Documents\GitHub\corefx\bin/AnyOS.AnyCPU.Debug/System.Runtime.CompilerServices.VisualC.Tests/netstandar
  d//RunTests.cmd C:\Users\hugh\Documents\GitHub\corefx\bin/testhost/netfx-Windows_NT-Debug-x64/
  Using C:\Users\hugh\Documents\GitHub\corefx\bin\testhost\netfx-Windows_NT-Debug-x64\ as the test runtime folder.
  Executing in C:\Users\hugh\Documents\GitHub\corefx\bin\AnyOS.AnyCPU.Debug\System.Runtime.CompilerServices.VisualC.Tes
  ts\netstandard\
  Running tests... Start time:  8:57:24.64
  xUnit.net Console Runner (64-bit Desktop .NET 4.0.30319.42000)
  System.IO.FileLoadException: Could not load file or assembly 'System.Runtime.CompilerServices.VisualC.Tests, Version=
  0.0.0.0' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference
  . (Exception from HRESULT: 0x80131040)
  Finished running tests.  End time= 8:57:30.50, Exit code = 1
C:\Users\hugh\Documents\GitHub\corefx\Tools\tests.targets(345,5): warning MSB3073: The command "C:\Users\hugh\Documents
\GitHub\corefx\bin/AnyOS.AnyCPU.Debug/System.Runtime.CompilerServices.VisualC.Tests/netstandard//RunTests.cmd C:\Users\
hugh\Documents\GitHub\corefx\bin/testhost/netfx-Windows_NT-Debug-x64/" exited with code 1. [C:\Users\hugh\Documents\Git
Hub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerServices.VisualC.Tests.csproj]
  The previous error was converted to a warning because the task was called with ContinueOnError=true.
  Build continuing because "ContinueOnError" on the task "Exec" is set to "true".
C:\Users\hugh\Documents\GitHub\corefx\Tools\tests.targets(353,5): error : One or more tests failed while running tests
from 'System.Runtime.CompilerServices.VisualC.Tests' please check C:\Users\hugh\Documents\GitHub\corefx\bin/AnyOS.AnyCP
U.Debug/System.Runtime.CompilerServices.VisualC.Tests/netstandard/testResults.xml for details! [C:\Users\hugh\Documents
\GitHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerServices.VisualC.Tests.csproj]
Done Building Project "C:\Users\hugh\Documents\GitHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.R
untime.CompilerServices.VisualC.Tests.csproj" (BuildAndTest target(s)) -- FAILED.


Build FAILED.

"C:\Users\hugh\Documents\GitHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerService
s.VisualC.Tests.csproj" (BuildAndTest target) (1) ->
(RunTestsForProject target) ->
  C:\Users\hugh\Documents\GitHub\corefx\Tools\tests.targets(345,5): warning MSB3073: The command "C:\Users\hugh\Documen
ts\GitHub\corefx\bin/AnyOS.AnyCPU.Debug/System.Runtime.CompilerServices.VisualC.Tests/netstandard//RunTests.cmd C:\User
s\hugh\Documents\GitHub\corefx\bin/testhost/netfx-Windows_NT-Debug-x64/" exited with code 1. [C:\Users\hugh\Documents\G
itHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerServices.VisualC.Tests.csproj]


"C:\Users\hugh\Documents\GitHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerService
s.VisualC.Tests.csproj" (BuildAndTest target) (1) ->
(RunTestsForProject target) ->
  C:\Users\hugh\Documents\GitHub\corefx\Tools\tests.targets(353,5): error : One or more tests failed while running test
s from 'System.Runtime.CompilerServices.VisualC.Tests' please check C:\Users\hugh\Documents\GitHub\corefx\bin/AnyOS.Any
CPU.Debug/System.Runtime.CompilerServices.VisualC.Tests/netstandard/testResults.xml for details! [C:\Users\hugh\Documen
ts\GitHub\corefx\src\System.Runtime.CompilerServices.VisualC\tests\System.Runtime.CompilerServices.VisualC.Tests.csproj
]

    1 Warning(s)
    1 Error(s)
```